.github/workflows/UnitTests.yml

```diff
@@ -19,11 +19,13 @@ jobs:
               uses: actions/checkout@v2
               with:
                   submodules: false
+            - name: "Fetching master to compare"
+              run: git fetch --no-tags --prune --depth=1 origin +refs/heads/${{ github.base_ref }}:refs/remotes/origin/${{ github.base_ref }}
```

Comment:
What exactly is the outcome of this step? Does it mean if there's no difference the step will fail?

Reply:
"**What else?**
Github actions does not checkout master branch, so lerna cannot compare to it and do partial build, so I added a git fetch for the targeted branch"

Reply:
More explanations here -> https://github.com/actions/checkout/issues/118